src/qibolab/instruments/emulator/pulse_simulator.py

```diff
@@ -180,6 +206,32 @@ def play(
             "output_states": output_states,
         }
 
+        if execution_parameters.acquisition_type is AcquisitionType.INTEGRATION:
+            if self.readout_simulator_config is not None:
+                for index, ro_pulse in enumerate(ro_pulse_list):
```

Comment:
Is the enumeration required when `index` is unused?

Reply:
opps sorry, it's a leftover